introduction.md

```diff
@@ -5,4 +5,8 @@ Hello my name is Isgrimner.
 
 My favorite Code School path is Ruby.
 
-Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.
\ No newline at end of file
+Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.
+
+*Git
+*Javascript
+*Ruby
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?